src/main/kotlin/net/axay/kotlinkitapi/implementation/Archer.kt

```diff
@@ -0,0 +1,53 @@
+package net.axay.kotlinkitapi.implementation
+
+import net.axay.kotlinkitapi.api.Kit
+import net.axay.kotlinkitapi.api.KitProperties
+import org.bukkit.Material
+import org.bukkit.entity.AbstractArrow
+import org.bukkit.entity.Arrow
+import org.bukkit.entity.LivingEntity
+import org.bukkit.entity.Player
+import org.bukkit.event.entity.EntityShootBowEvent
+import org.bukkit.event.entity.ProjectileHitEvent
+import org.bukkit.inventory.ItemStack
+import org.bukkit.potion.PotionEffect
+import org.bukkit.potion.PotionEffectType
+import java.util.*
+import kotlin.math.roundToInt
+
+class ArcherProperties : KitProperties() {
+    val effectDuration by int(80)
+    val effectMultiplier by int(0)
+    val arrowAmount by int(6)
+    val likelihood by int(50)
```

Comment:
dou you mean "probability"?

Reply:
means the same

Reply:
yeah but please change it because it should be a consistent naming scheme, if a similar property will be available in other kits as well - and likelihood is very uncommon in programming, probability is better